### PR TITLE
Add staging and Ferran as sysadmin

### PR DESCRIPTION
**Pull request by enricostano**

### CODE REVIEW

File: file/keys/ferran.pub [hunk 1]
```diff
@@ -0,0 +1 @@
+ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIGz0AdSjKU0w0wjjAZ/PGMqbIPPxJmxUX/YQYrfgvtcK ferran
```
Comment (enricostano) [inline file/keys/ferran.pub]: @raneq :point_up_2:

File: inventory/host_vars/stagingtryton.somconnexio.coop/config.yml [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    state: present
+  - name: enrico
+    ssh_key: '{{ inventory_dir }}/../file/keys/enrico.pub'
+    state: present
+  - name: antonio
+    ssh_key: '{{ inventory_dir }}/../file/keys/antonio.pub'
+    state: present
+  - name: ferran
+    ssh_key: '{{ inventory_dir }}/../file/keys/ferran.pub'
+    state: present
```
Comment (enricostano) [inline inventory/host_vars/stagingtryton.somconnexio.coop/config.yml]: @raneq :point_up_2:

File: file/keys/ferran.pub (hunk 1 above)
Comment (raneq) [reply file/keys/ferran.pub]: correct!

File: inventory/host_vars/stagingtryton.somconnexio.coop/config.yml (hunk 2 above)
Comment (raneq) [reply inventory/host_vars/stagingtryton.somconnexio.coop/config.yml]: this is coherent with the other entries and the filename of my public key.

**Pull request merged**